rocky/rocky/views/tasks.py

```diff
@@ -115,3 +118,88 @@ def get_context_data(self, **kwargs):
         ]
 
         return context
+
+
+class AllTaskListView(ListView):
+    paginate_by = 100
+    paginator_class = RockyPaginator
+
+    # @staff_member_required
+    def get_queryset(self):
+        task_type = self.request.GET.get("type", self.plugin_type)
+        status = self.request.GET.get("scan_history_status") if self.request.GET.get("scan_history_status") else None
+        input_ooi = self.request.GET.get("scan_history_search") if self.request.GET.get("scan_history_search") else None
+
+        if self.request.GET.get("scan_history_from"):
```

Comment:
```suggestion
        if "scan_history_from" in self.request.GET:
```

Reply:
Isn't there a difference between these checks? I think the first one also checks the value of the key, while your suggestion only checks existence of the key. In any case; I copied it from your `TaskListView` ;)